app/src/components/dialog/SubmitBiohubDialog.tsx

```diff
@@ -1,38 +1,26 @@
-import { CircularProgress, DialogContent } from '@material-ui/core';
-import Box from '@material-ui/core/Box';
+import { DialogContent, DialogContentText } from '@material-ui/core';
 import Button from '@material-ui/core/Button';
 import Dialog from '@material-ui/core/Dialog';
 import DialogActions from '@material-ui/core/DialogActions';
 import DialogTitle from '@material-ui/core/DialogTitle';
-import makeStyles from '@material-ui/core/styles/makeStyles';
 import useTheme from '@material-ui/core/styles/useTheme';
 import useMediaQuery from '@material-ui/core/useMediaQuery';
-import { Formik, FormikProps } from 'formik';
-import React, { useRef, useState } from 'react';
+import LoadingButton from 'components/buttons/LoadingButton';
+import { SubmitBiohubI18N } from 'constants/i18n';
+import { DialogContext } from 'contexts/dialogContext';
+import { Formik, FormikProps, FormikValues } from 'formik';
+import React, { PropsWithChildren, useContext, useRef, useState } from 'react';
 import yup from 'utils/YupSchema';
-import { ErrorDialog } from './ErrorDialog';
-
-const useStyles = makeStyles((theme) => ({
-  wrapper: {
-    position: 'relative'
-  },
-  buttonProgress: {
-    color: theme.palette.primary.main,
-    position: 'absolute',
-    top: '50%',
-    left: '50%',
-    marginTop: -12,
-    marginLeft: -12
-  }
-}));
+import ComponentDialog from './ComponentDialog';
+import { IErrorDialogProps } from './ErrorDialog';
 
 /**
  *
  *
  * @export
  * @interface ISubmitBiohubDialogProps
  */
-export interface ISubmitBiohubDialogProps {
+export interface ISubmitBiohubDialogProps<V> {
```

Comment:
nit: can we rename `V` to `InitialValues` or something along those lines

Reply:
Or I guess the "initial" part is irrelevant, `FormikValues` maybe?

Reply:
Oops, sure

Reply:
I went for just `Values`, since `FormikValues` is the interface that this type extends